templates/http/default.conf.j2

```diff
@@ -3,7 +3,9 @@
 {% if item.value.upstreams is defined and item.value.upstreams %}
 {% for upstream in item.value.upstreams %}
 upstream {{ item.value.upstreams[upstream].name }} {
+{% if item.value.upstreams[upstream].lb_method is defined %}
```

Comment:
While at it, can you add a check for it being defined and not an empty string?

Reply:
Good point. I've added the check for emptiness.